INSTALL.md

```diff
@@ -84,13 +84,20 @@ If you are interested in packaging, documenting, or contributing otherwise, plea
 
 ## Docker images <a name="docker"></a>
 You can run Shogun in [our own cloud](cloud.shogun.ml) or set up your own using our
-[Docker images](https://hub.docker.com/r/shogun/shogun-dev/) as:
+[Docker images](https://hub.docker.com/r/shogun/shogun/) as:
 
-    sudo docker pull shogun/shogun
-    sudo docker run -it shogun/shogun bash
+    sudo docker pull shogun/shogun:master
+    sudo docker run -it shogun/shogun:master bash
 
 We offer images for both the latest release and nightly development builds.
 
+for developer version [Docker images](https://hub.docker.com/r/shogun/shogun-dev/)
+
+    sudo docker pull shogun/shogun-dev
+    sudo docker run -it shogun/shogun-dev bash
+
+Go to the docker image page and then to build details tab before downloading docker image to check if build is sucessfull, otherwise you might run into errors when running the docker image. If you didn't understand this last part ignore it(check docker documentation if you want to learn more).
```

Comment:
I'd make this: (shorter, better structure and some corrected typos)

"Check the "details" tab before downloading to check if the latest build was successful (otherwise you might run into errors when running the docker image)."